### PR TITLE
vuln(NSWG-ECO-490): node.extend

### PR DESCRIPTION
**Pull request by lirantal**
/cc @Elexy @ljharb

### CODE REVIEW

File: vuln/npm/490.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "name": "asgerf",
+    "website": null,
+    "username": "asgerf"
+  },
+  "module_name": "node.extend",
+  "cves": [
+    "CVE-2018-16491"
+  ],
+  "vulnerable_versions": "<2.0.1 || <1.1.7",
+  "patched_versions": ">=1.1.7 <2.0.1 || >= 2.0.1",
```
Comment (nschonni) [inline vuln/npm/490.json]: should the `<2.0.1` be removed from the range? I seems to overlap the following `||`

Comment (ljharb) [reply vuln/npm/490.json]: Why not `^1.1.7 || >= 2.0.1`?

Comment (lirantal) [reply vuln/npm/490.json]: That works too, I'll update.

File: vuln/npm/490.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "website": null,
+    "username": "asgerf"
+  },
+  "module_name": "node.extend",
+  "cves": [
+    "CVE-2018-16491"
+  ],
+  "vulnerable_versions": "<2.0.1 || <1.1.7",
+  "patched_versions": "^1.1.7 || >= 2.0.1",
+  "recommendation": "Update node.extend module to version >=1.1.7 <2.0.1 || >= 2.0.1",
```
Comment (nschonni) [inline vuln/npm/490.json]: The version range in here should probably updated too

Comment (lirantal) [reply vuln/npm/490.json]: yes thanks Nick!

**Comment by Elexy** [comment]
@lirantal the cvss vector score seems to be set now in H1. Anything else I can do?

**Comment by lirantal** [comment]
@Elexy I updated the score and vector to reflect here.

**Pull request merged**